tuna/worker_interface.py

```diff
@@ -454,10 +488,10 @@ def reset_job_state(self):
       except queue.Empty:
         break
 
-  def run(self):
+  def run(self) -> Optional[bool]:  #type: ignore[override]
```

Comment:
why ignore?

Reply:
run() return type is "None" in supertype "BaseProcess"